migrations/sql/V2018.02.20.0900__AddSmallCountryColumnToBurdenEstimate.sql

```diff
@@ -0,0 +1,259 @@
+ALTER TABLE burden_estimate
+  ADD COLUMN country_nid SMALLINT NULL;
+
+UPDATE burden_estimate SET country_nid=4 WHERE country='AFG';
```

Comment:
I'm sure you've go this right, but this does make me nervous.  Can we do this with a join instead?

Reply:
I took these values from the original migration that inserted the countries... we could, but again just trying to make it more efficient